File: app/src/main/java/org/autojs/autojs/ui/doc/DocumentationActivity.java
package org.autojs.autojs.ui.doc;

import android.os.Bundle;
import android.webkit.WebView;

import org.autojs.autojs.Pref;
import org.autojs.autojs.R;
import org.autojs.autojs.databinding.ActivityDocumentationBinding;
import org.autojs.autojs.ui.BaseActivity;

public class DocumentationActivity extends BaseActivity {

    public static final String EXTRA_URL = "url";
    private ActivityDocumentationBinding binding;
    private WebView mWebView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityDocumentationBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        setToolbarAsBack(getString(R.string.text_tutorial));
        mWebView = binding.ewebView.getWebView();
        String url = getIntent().getStringExtra(EXTRA_URL);
        if (url == null) {
            url = Pref.getDocumentationUrl() + "index.html";
        }
        mWebView.loadUrl(url);
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        binding = null;
    }

    @Override
    public void onBackPressed() {
        if (mWebView.canGoBack()) {
            mWebView.goBack();
        } else {
            super.onBackPressed();
        }
    }
}

File: app/src/main/java/org/autojs/autojs/ui/doc/DocumentationActivity.kt
package org.autojs.autojs.ui.doc

import android.os.Bundle
import org.autojs.autojs.Pref
import org.autojs.autojs.R
import org.autojs.autojs.ui.BaseActivity
import org.autojs.autojs.ui.widget.EWebView

class DocumentationActivity : BaseActivity() {

    private lateinit var eWebView: EWebView
    private lateinit var webView: android.webkit.WebView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_documentation)
        eWebView = findViewById(R.id.eweb_view)
        setUpViews()
    }

    private fun setUpViews() {
        setToolbarAsBack(getString(R.string.text_tutorial))
        webView = eWebView.webView
        val url = intent.getStringExtra(EXTRA_URL) ?: "${Pref.getDocumentationUrl()}index.html"
        webView.loadUrl(url)
    }

    override fun onBackPressed() {
        if (webView.canGoBack()) {
            webView.goBack()
        } else {
            super.onBackPressed()
        }
    }

    companion object {
        const val EXTRA_URL = "url"
    }
}
